.github/workflows/cypress.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           install: false
           # we launch the app in preview mode to avoid issues with hmr websockets from vite polluting the mocks
           start: yarn preview:test
-          browser: chrome
+          # browser: chrome
```

Comment:
Does that change anything? Less buggy?

Reply:
@pyphilia there is an open issue with cypress (actually a bug in chrome) that made the tests fail. See #806 Removing this uses the default for cypress which is Electron (i know not the best but at least it works). #807 should solve the issue with cypress.